qiskit/transpiler/target.py

```diff
@@ -627,6 +667,25 @@ def check_obj_params(parameters, obj):
         if operation_name in self._gate_map:
             if parameters is not None:
                 obj = self._gate_name_map[operation_name]
+                if inspect.isclass(obj):
+                    warnings.warn(
+                        "A parameter was specified for an operation that is defined as a "
+                        "globally supported class there is no available validation (including "
+                        "whether the specified operation supports parameters), the returned "
+                        "value will not factor in the argument `parameters`.",
```

Comment:
This sentence reads weirdly to me - I think it might be missing some words / a sentence break in the middle?

Reply:
I started to reword this to clean it up, but I'm actually thinking we should just drop this warning, something I've learned the hardware is warning at what is nominally an internal compiler data structure tends to not be actionably by the majority of users. For someone like me who is working on new compiler techniques and testing the compiler the target API is user facing, but for most people calling `transpile()` they never see this as it's internal between the backend and the compiler. So if a backend defines a variable width gate that has parameters this could trigger through no direct action from the user.

Reply:
Converted this to a comment (and reworded) in: https://github.com/Qiskit/qiskit-terra/pull/8852/commits/07ed34fb046ec7fb798406405c55aaffaf476535